CONTRIBUTING.md

```diff
@@ -3,10 +3,10 @@ Thank you for your interest in contributing! Because we're a small team, we have
 
 ## Issues
 1. Issues are the best place to propose a new feature. Keep in mind that htmx is a small library, so there are lots of great ideas that don't fit in the core; it's always best to check in about an idea before doing a bunch of work on it.
-1. When proposing a new features, we will often suggest that you implement it as an [extension](https://htmx.org/extensions), so try that first. Even if we don't end up supporting it officially, you can publish it yourself and we can link to it.
+1. When proposing a new features, we will often suggest that you implement it as an [extension](https://htmx.org/docs/#extensions), so try that first. Even if we don't end up supporting it officially, you can publish it yourself and we can link to it.
 1. Search the issues before proposing a feature to see if it is already under discussion. Referencing existing issues is a good way to increase the priority of your own.
 1. We don't have an issue template yet, but the more detailed your description of the issue, the more quickly we'll be able to evaluate it.
-1. See an issue that you also have? Give it a reaction (and comment, if you have something to add). We note that!
+1. Do you see an issue that you have? Give it a reaction (and comment, if you have something to add). We note that!
```

Comment:
Not sure about this one to be honest
- I feel dropping the "also" makes it lose a bit of its meaning/context (as we're just using the generic "issue" word here and not specifically "GitHub issue", even though that is what is implied)
- The "do you" wording may be a bit redundant/unnecessary, or at least "less direct" if that makes sense. Btw, 2 lines below we have a similar wording with "want to contribute [...] ?", which would illustrate even more what I mean if it were to be changed to match this new wording style, as there are 2 occurrences in the same sentence

Reply:
Thanks for taking the time to share your thoughts.

Perhaps you or the original author used a casual language and that's what you're preferring. I find it more direct to have a language of Do you and using reference to issues, we could clarify further Github issues.

Basically removing ambiguity and having calls for action.

I'm not married to this proposal.

Reply:
> I'm not married to this proposal.

Me neither! Disclaimer: I didn't write this, and I'm not a native english speaker, this was just my personal opinion.

As for the PR itself, I would probably suggest only pushing the link fix in this one, and why not make another PR with the wording change, just so that I can merge the fix link right away, while we let the wording being discussed on its own PR

Reply:
I agree fully with the last bit. I thought of it as I was editing the doc and typical dev manner stepped into to add just one more change, haha. Thank you.

Even if you arent a native speaker, your input is still valid and appreciated.

Reply:
I just made an update patch addressing both issues. thanks